src/components/PostList/PostList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PostItem from '../PostItem/PostItem';
+import { IPost } from '../../models/IPosts';
+import {
+  useDeletePostMutation,
+  useGetPostsQuery,
+  useUpdatePostMutation
+} from '../../services/ForumService';
+import Loader from '../Loader/Loader';
+
+const PostList = () => {
+  const { data: posts, isLoading } = useGetPostsQuery(1);
+  const [deletePost] = useDeletePostMutation();
+  const [updatePost] = useUpdatePostMutation();
+
+  const handleRemove = async (id: number) => {
+    deletePost(id)
+  }
```

Comment:
❗️
Лучше придерживаться единого нейминга
```suggestion
  const handleDelete = async (id: number) => {
    deletePost(id)
  }
```

Reply:
придержуюсь одного нейминга! Готово!